app/views/layouts/application.html.erb

```diff
@@ -21,12 +21,11 @@
   <meta name="description" content="<%= t :mumuki_short_description %>"/>
   <%= stylesheet_link_tag 'application', media: 'all', 'data-turbolinks-track': 'reload' %>
   <%= csrf_meta_tags %>
-  <script type="text/javascript">
-    window.mumukiLocale = <%= raw Organization.current.locale_json %>;
-  </script>
   <%= javascript_include_tag 'application', 'data-turbolinks-track': 'reload' %>
   <script type="text/javascript" src="<%= extension_javascript_url %>" defer data-turbolinks-track="reload"></script>
   <script type="text/javascript">
+    window.mumukiLocale = <%= raw Organization.current.locale_json %>;
+    mumuki.locale = '<%= Organization.current.locale %>';
```

Comment:
We should refactor this code eventually. It should be in another file, its logic is repeated and I don't think `mumukiLocale` and `mumuki.locale`  are both necessary